crates/sui-graphql-e2e-tests/tests/event_connection/event_connection.move

```diff
@@ -259,8 +247,23 @@ module Test::M2 {
       type {
         repr
       }
-      senders {
-        address
+      json
+      bcs
+    }
+  }
+}
+
+//# run-graphql --variables B
+{
+  eventConnection(
+    filter: {sender: $B}
```

Comment:
Can you use interpolation instead of variables? It'll save me a merge conflict later.